src/main/scala/cromwell/engine/WorkflowManager.scala

```diff
@@ -58,6 +60,13 @@ trait ActorWorkflowManager extends WorkflowManager {
   }
 
   override def workflowStatus(id: WorkflowId): Option[WorkflowState] = workflowById(id) map {x => workflowState(x)}
+  override def workflowOutputs(id: WorkflowId): Map[FullyQualifiedName, WdlValue] = {
+    val outputs = workflowById(id) map {_ ! GetOutputs}
```

Comment:
I wasn't sure about this.  I thought from the conversation yesterday with @geoffjentry that this was the right way to do it... what is the right way to do it?